tutorials/migrating/upgrading_to_godot_4.2.rst

```diff
@@ -0,0 +1,247 @@
+Method ``get_format`` changes return type from ``uint32`` to ``uint64``                                                   |✔️|                 |❌|                  |❌|                  `GH-81138`_
+**RenderingDevice**
+Enum field ``BarrierMask.BARRIER_MASK_RASTER`` changes value from ``1`` to ``9``                                          |✔️|                 |✔️|                  |✔️|                  `GH-79911`_
+Enum field ``BarrierMask.BARRIER_MASK_ALL_BARRIERS`` changes value from ``7`` to ``32767``                                |✔️|                 |✔️|                  |✔️|                  `GH-79911`_
+Enum field ``BarrierMask.BARRIER_MASK_NO_BARRIER`` changes value from ``8`` to ``32768``                                  |✔️|                 |✔️|                  |✔️|                  `GH-79911`_
+Method ``shader_create_from_bytecode`` adds a new ``placeholder_rid`` optional parameter                                  |✔️|                 |✔️ with compat|      |✔️|                  `GH-79606`_
+Method ``shader_get_vertex_input_attribute_ask`` changes return type from ``uint32`` to ``uint64``                        |✔️|                 |❌|                  |❌|                  `GH-81138`_
+**SurfaceTool**
+Method ``commit`` changes ``flags`` parameter type from ``uint32`` to ``uint64``                                          |✔️|                 |✔️ with compat|      |✔️ with compat|      `GH-81138`_
+========================================================================================================================  ===================  ====================  ====================  ===========
+
+Text
+^^^^
+
+====================================================================================================================================  ===================  ====================  ====================  ===========
+Change                                                                                                                                GDScript Compatible  C# Binary Compatible  C# Source Compatible  Introduced
+====================================================================================================================================  ===================  ====================  ====================  ===========
+**Font**
+Method ``set_fallbacks`` replaced with ``fallbacks`` property                                                                         |✔️|                 |❌|                  |❌|                  `GH-78266`_
+Method ``get_fallbacks`` replaced with ``fallbacks`` property                                                                         |✔️|                 |❌|                  |❌|                  `GH-78266`_
+Method ``find_variation`` adds new ``spacing_top``, ``spacing_bottom``, ``spacing_space`` and ``spacing_glyph`` optional parameters   |✔️|                 |✔️ with compat|      |✔️|                  `GH-80954`_
```

Comment:
```suggestion
Method ``find_variation`` adds new ``spacing_top``, ``spacing_bottom``, ``spacing_space``, and ``spacing_glyph`` optional parameters   |✔️|                 |✔️ with compat|      |✔️|                  `GH-80954`_
```
Please use [Oxford comma](https://docs.godotengine.org/en/latest/contributing/documentation/docs_writing_guidelines.html#use-the-oxford-comma-to-enumerate-anything).